tools/stress-cluster/services/Stress.Watcher/publish.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+    [string] $Tag = "latest",
+    [bool] $Login = $true
+)
+
+docker build . -t stresswatcher
+docker tag stresswatcher stresstestregistry.azurecr.io/services/stresswatcher:$Tag
+docker tag stresswatcher stressprodregistry.azurecr.io/services/stresswatcher:$Tag
+
+az acr login -n stresstestregistry
+docker push stresstestregistry.azurecr.io/services/stresswatcher:$Tag
+
+#az acr login -n stressprodregistry
```

Comment:
Do you plan to control this via a flag later?

Reply:
Actually this shouldn't be commented out, I was just testing.

But more generally, yeah there are now a couple different components to be deployed for the cluster. I think I'm going to wrap it all up in one script+pipeline, and then optionally trigger it on merge, so the logic is in one place.